src/rp2_common/pico_crt0/rp2040/memmap_copy_to_ram.ld

```diff
@@ -66,7 +66,8 @@ SECTIONS
         KEEP (*(.embedded_block))
         __embedded_block_end = .;
         KEEP (*(.reset))
-    }
+        . = ALIGN(4);
+    } > FLASH
```

Comment:
This change was ported here to align with the implementation found in the rp2350/memmap_copy_to_ram.ld version of this file.